nevergrad/optimization/base.py

```diff
@@ -456,7 +456,7 @@ def ask(self) -> p.Parameter:
                     candidate = self.parametrization.spawn_child()
             if candidate.satisfies_constraints():
                 break  # good to go!
-            if self._penalize_cheap_violations:
+            if self._penalize_cheap_violations or self.no_parallelization:
```

Comment:
with non-parallelizable optimizer, we need to provide a tell

Reply:
waow congrats...